ow_power_system/config/system.cfg

```diff
@@ -27,17 +27,20 @@ voltage_range: 0.1			        # volts
 # are based on a Europa Lander draft specification, others on sample
 # values from other planetary rovers, and some are just guesses.
 
-# Baseline values (watts) for constant power draws.
-baseline_power_camera: 0.1
-baseline_power_communications: 0.6
+# (per-cell Baseline power draws (Watts)
 baseline_power_computing: 0.1
 baseline_power_heating: 1.2
 baseline_power_science_instr: 0.6
 baseline_power_sample_handling: 0.6
 baseline_power_other: 2.2
 
-# Power coefficients for dynamically computed power draw.
-wattage_coefficient_lights: 0.3    # power draw (W) per light at max intensity
+# Baseline power draws (Watts)
+power_baseline_camera_controller: 1.6  # power draw of the camera controller
```

Comment:
I like the new naming convention.  Small request: change the other `baseline_power*` entries to match this one.

Reply:
> I like the new naming convention. Small request: change the other `baseline_power*` entries to match this one.

Can we do that in the hotfix that will make those baselines distribute across all batteries? I like that they're differentiated right now, since they currently behave differently.